### DISCUSSION TITLE
Swipe typing issue for spanish language

### DISCUSSION THREAD

**Discussion by Maximiliem** [florisboard/florisboard#2577] [Feedback]
When I try to write in spanish with swipe typing, the keyboard recognizes only English words, so I can't use this feature to write in spanish.
I don't know if this happens with other languages.
Thanks!

**Comment by patrickgold** [comment]
Glide typing is not supported in the latest v0.4 release and will be a major goal for v0.5 or v0.6. The old implementation only supported English US.
